documentation/modules/security/proc-recovering-expired-ca-certs.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='proc-recovering-expired-ca-certs-{context}']
+
+= Manually recovering from expired Cluster Operator-managed CA certificates
+
+[role="_abstract"]
+Cluster and clients CA certificates are automatically renewed by the Cluster Operator at the start of their renewal periods. 
+However, under certain circumstances, the renewal process may fail due to operational issues or disruptions. 
```

Comment:
Should we add some reasonably-sounding examples? This sounds a bit too much like `the software is sh*t and it does not work`. ;-)

Maybe these two:
* When the operator is not running for a longer time
* When the Kafka cluster is not ready for some other reason and the operator cannot renew the CA

=> these are based on the situations when this comes up.

Reply:
Changed to:

> The Cluster Operator automatically renews the Cluster and Clients CA certificates when their renewal periods begin. 
> Nevertheless, unexpected operational problems or disruptions may prevent the renewal process, such as prolonged downtime of the Cluster Operator or unavailability of the Kafka cluster. 
> Promptly addressing this situation by recovering the deployment from using expired certificates ensures ongoing secure communication.

Reply:
see next comment for further adjustment